Makefile.build

```diff
@@ -630,15 +627,73 @@ test-suite: world byte $(ALLSTDLIB).v
 # Default rules for compiling ML code
 ###########################################################################
 
-# Target for libraries .cma and .cmxa
+gramlib__pack:
+	mkdir -p $@
+
+# gramlib.ml contents
+gramlib__pack/gramlib.ml: | gramlib__pack
+	echo " \
+module Ploc    = Gramlib__Ploc \
+module Plexing = Gramlib__Plexing \
+module Gramext = Gramlib__Gramext \
+module Grammar = Gramlib__Grammar" > $@
```

Comment:
Why can't this file just be there, along with its containing directory?
In some sense this plays the role of the missing (why btw) `gramlib/gramlib.mllib` (or `.mlpack`).

Reply:
Indeed would the make system be adapted so .mlpack packs in this way, this file would be auto-generated, so I think it is better to have it this way; the file is not needed in the regular `gramlib` tree so it should be like this then.